src/server/editorServices.ts

```diff
@@ -1674,10 +1674,12 @@ namespace ts.server {
             if (!this.eventHandler || this.suppressDiagnosticEvents) {
                 return;
             }
+            const diagnostics = project.getLanguageService().getCompilerOptionsDiagnostics();

```

Comment:
concat?

Reply:
using same array since gets a new array so we can just use same one.